hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/CommonUtils.java

```diff
@@ -88,4 +136,65 @@ public static boolean productWouldOverflow(final long multiplier, final long mul
     static void setSha384HashTag(final String sha384HashTag) {
         CommonUtils.sha384HashTag = sha384HashTag;
     }
+
+    /**
+     * check TransactionBody and return the HederaFunctionality. This method was moved from MiscUtils.
```

Comment:
`This method was moved from MiscUtils` <- This is not really a needed information in JavaDoc.

Reply:
@Neeharika-Sompalli asked me to add this. The important one is next sentence: NODE_STAKE_UPDATE is not checked in this method, since it is not a user transaction.